src/test/java/ay2021s1_cs2103_w16_3/finesse/testutil/TransactionUtil.java

```diff
@@ -26,6 +28,13 @@ public static String getAddCommand(Transaction transaction) {
         return AddCommand.COMMAND_WORD + " " + getTransactionDetails(transaction);
     }
 
+    /**
+     * Returns an add expense command string for adding the {@code transaction}.
```

Comment:
`transaction` is no longer the parameter name
```suggestion
     * Returns an add expense command string for adding the {@code expense}.
```

Reply:
This was left as such to mirror the corresponding change in #56. Can I confirm this is not intended?